code/modules/assembly/anomalies.dm

```diff
@@ -173,6 +198,7 @@
 /obj/item/assembly/signaler/anomaly/tvstatic
 	name = "\improper static anomaly core"
 	desc = "The stabilized core of a static anomaly. Your head hurts just staring at it"
+	grind_results = list(/datum/reagent/three_eye = 5)
```

Comment:
erika what the fuck. thats Three Eye.

Reply:
yeah i thought it'd be cool for the anomaly that gives you brain damage.

Reply:
its funny but i dont think you should use the Admin Meme Chemical That Unavoidably Kills You And Makes You See Green. idk im not a maintainer.

Reply:
It literally is permanent and cannot be undone, even by aheals. Not a good idea

Reply:
a: who the fuck ported this
b: who the fuck let this stay in here this long

Reply:
yeah okay so i went and read the code and it only does that if you overdose. which is stupid. but something to address in another pr. you'd need to find 5 static anomalies to get enough to overdose yourself like this

Reply:
also the code is broken its not supposed to turn you into the lime man forever.

Reply:
okay so - i tested it and it did clear itself. idk what about overdose makes you turn into a citrus fruit, but unless a maintainer says nuh uh I think this is okay.
It's a rarer anomaly and a small amount of what is ostensibly a drug

Reply:
the funny :+1: